src/parser/job.rs

```diff
@@ -201,35 +218,58 @@ impl Job {
         }
     }
 
-    pub fn restart(self: &mut Self, job_name: &String) {
+    pub fn restart(self: &mut Self, job_name: &String, target_process: Option<u32>) {
         println!("log: restart {}", job_name);
-        self.stop(job_name);
-        self.start(job_name);
+        self.stop(job_name, None);
```

Comment:
stop does'nt need target_process ?

Reply:
Arent we supposed to stop a targetted processes instead of all process ?

Reply:
is it just for start ?

Reply:
Ah okay I understood the comment yes I agree